source/_docs/configuration/state_object.markdown

```diff
@@ -1,33 +1,59 @@
+| `state.domain`        | Domain of the entity. Example: `light`.                                                                                                                                                         |
+| `state.object_id`     | Object ID of entity. Example: `kitchen`.                                                                                                                                                        |
+| `state.name`          | Name of the entity. Based on `friendly_name` attribute with fall back to object ID. Example: `Kitchen ceiling`.                                                                                 |
+| `state.last_changed`  | Time the state changed in the state machine in UTC time. This is not updated if only state attributes change. Example: `2013-09-17 07:32:51.715874+00:00`.                                      |
+| `state.last_reported` | Time the state was written to the state machine in UTC time. This timestamp is updated regardless of any changes to the state or state attributes. Example: `2013-09-17 07:32:51.715874+00:00`. |
+| `state.last_updated`  | Time the state or state attributes changed in the state machine in UTC time. This is not updated if neither state nor state attributes changed. Example: `2013-09-17 07:32:51.715874+00:00`.    |
+| `state.attributes`    | A dictionary with extra attributes related to the current state.                                                                                                                                |
+| `state.context`       | A dictionary with extra attributes related to the context of the state.                                                                                                                         |
+
+### About the state
+
+The screenshot shows three lights in different states (the `state.state`): `on`, `off`, and `unavailable`. Each light comes with its own entity state attributes such as `supported_color_modes`, `supported_features`. These attributes have their own state: the state of `supported_color_modes` is `color_temp` and `hs`, the state of the `supported_features` attribute is `4`.
+
+<p class='img'>
+  <img src='/images/integrations/light/state_light.png' alt='Screenshot showing three lights with different states: `on`, `off`, or `unavailable`'>
+  Three lights with different states: `on`, `off`, or `unavailable`.
+</p>
+
+The `state.state` is the heart of the [state object](#about-the-state-object). State holds the information of interest of an entity. For example, if a
+light is on or off, the current temperature, or the amount of energy used. The state object stores 3
+timestamps related to the state: `last_updated`, `last_changed`, and `last_reported`. Each
+entity has exactly one state, and the state only holds one value at a time.
```

Comment:
I guess we could mention here that the automation state trigger only triggers on `state.state` here?

Reply:
The state trigger fires on both the main state and the state attributes, depending on how the state trigger is configured. What do mean exactly?

Reply:
I think I got that wrong. I thought that the problem with using event entities and selecting the "Event type" as attribute and "To" state (e.g. "Pressed once") doesn't work is because the trigger doesn't fire. But it is not that it doesn't trigger per-se. It is that if the user does a "Pressed once" two times in a row, there is no state change, so it won't trigger.

So yeah, scratch that.

Reply:
Yeah, if the user configures the state trigger to trigger on an attribute change, and that attribute doesn't change, the state trigger won't fire, even if the main state changes.

Reply:
Ah right, the main state even changes to a new date for event entites. But since it is filtered by attribute, that change doesn't "count".